pkg/resource/acl/custom_set_output.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package acl
+
+import (
+	"context"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+	svcutil "github.com/aws-controllers-k8s/memorydb-controller/pkg/util"
+)
+
+func (rm *resourceManager) customDescribeACLsSetOutput(
+	ctx context.Context,
+	r *resource,
+	resp *svcsdk.DescribeACLsOutput,
+	ko *svcapitypes.ACL,
+) (*svcapitypes.ACL, error) {
+	if len(resp.ACLs) == 0 {
+		return ko, nil
+	}
```

Comment:
Since `resp` is only used here. I think this check should go to the describe hook. which can reduce the number of parameters needed for this function:
```
func (rm *resourceManager) customDescribeACLsSetOutput(
	ctx context.Context,
	r *resource,
) ([]*events..., error) {
```

Reply:
Same as above.

Reply:
Moved to `sdk_read_many_post_set_output.go.tpl`.